### ISSUE TITLE
KMS ENCRYPTION

### ISSUE THREAD

**Issue by ahmadsn26** [clarkie/dynogels#104]
As the title says it all 

is it possible to add a feature to pass on our kms key while initiating dynogels and then the data is encrypted whenever we call create or update other then the primary key attribute and on the get method we get the decrypted data this way we wont have to make a centralized encryption and decryption methodology and will just be able to encrypt and decrypt our data base on our needs

**Comment by clarkie** [comment]
Yeah, sounds good. I'd be happy to look at a PR that implements this.

Thanks

**Comment by dangerginger** [comment]
I don't know too much about this.  @ahmadsn26, if you can provide more detail on how this should work then maybe you or someone else would be willing to submit a pull request.  If not I think we should close this issue for lack of interest.
